docs/release/trg-5/trg-5-6.md

```diff
@@ -0,0 +1,32 @@
+---
+title: TRG 5.06 - Application Configuration
+---
+
+| Author               | Status | Created     | Post-History    |
+|----------------------|--------|-------------|-----------------|
+| Catena-X System Team | Active | 30-Nov-2022 | Initial release |
+
+## Description
+
+Every configuration aspect of your application has to be configurable for your Helm Chart.
```

Comment:
Perhaps we can say "has to be configurable through your helm chart"?

Reply:
Check.